ovirt-guest-agent-fedora/ovirt-container-hibernate.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+/usr/bin/nsenter --mount=/proc/1/ns/mnt -- /usr/share/ovirt-guest-agent/hibernate $@
```

Comment:
I've addressed that in #37 by dropping the wrapper as it is really unnecessary here